project_task_subtask/doc/index.rst

```diff
@@ -0,0 +1,19 @@
+=====================
+ Project task subtask
+=====================
+
+Usage
+=====
+
+* new tab "Subtasks" at task's form
+* when new subtask is created\changed message is sent to user that assigned to this subtask
+* delete subtask can only user who create it
+* Each subtask has button to swith state:
+ * DONE
+ * TODO
+ * CANCELED
+
+* new menu "Subtasks"
+ * default filter: "My", "TODO"
+ * optional group by: "Project", "Task"
```

Comment:
Move to README and add a usage description (Open menu, click, login as another user etc)